src/main/java/hw4/Task_6.java

```diff
@@ -0,0 +1,18 @@
+package hw4;
+
+public class Task_6 {
+    public static void main(String[] args) {
+
+        int[] numbers = {10, 5, -5, 6, 15, -23, 50, -17, 20, -3, 32, 18, 2, -50, 24, -42, 7, 6, 4, 11, -36, 36, -42, 42, 28, -28, 8, -8, 9, -9, 12, -12, -13, 13, 6, 4, 2, 4, 47, 49, -4, 6, 1, 3, 9, 11, 12, 14};
```

Comment:
Use random here like in task 5. We've discussed how to generate random number in range. You can find it here
https://dan-it.gitlab.io/fs-book-ua/java-basic/ext/ext_generation_random_number/ext_generation_random_number.html